packages/contracts-periphery/contracts/L1/L1ERC721Bridge.sol

```diff
@@ -1,9 +1,10 @@
 // SPDX-License-Identifier: MIT
 pragma solidity 0.8.15;
 
+import { ERC721Bridge } from "../universal/op-erc721/ERC721Bridge.sol";
 import {
     CrossDomainEnabled
-} from "@eth-optimism/contracts/contracts/libraries/bridge/CrossDomainEnabled.sol";
+} from "@eth-optimism/contracts/libraries/bridge/CrossDomainEnabled.sol";
```

Comment:
Don't think we need this import as it's already in `ERC721Bridge.sol`

Reply:
The `CrossDomainEnabled` import is removed in [this PR](https://github.com/ethereum-optimism/optimism/pull/3491/files), so I think it's fine to just leave this as-is here.